app/views/content/life-as-a-teacher/biology/_article.html.erb

```diff
@@ -0,0 +1,103 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach biology</h2>
+    <div class="inset">
+      <p>Biology is the study of living things, the intricate systems that sustain them, and the delicate balance of our ecosystems. As a biology teacher, you'll have the opportunity to inspire the next generation to explore the wonders of life itself.</p>
+      <p>In addition to the personal rewards of teaching biology, there are practical benefits too, including:</p>
+      <ul>
+        <li>a <a href="/life-as-a-teacher/teacher-pay-and-benefits">competitive salary starting at <%= value :salaries_starting_minshortened %></a> (or higher in London)</li>
+        <li>a <a href="/life-as-a-teacher/what-pension-does-a-teacher-get">generous and secure pension</a></li>
```

Comment:
This link needs updating now that the Pensions page has moved (causing a build failure)